types/secure-random-password/index.d.ts

```diff
@@ -1,20 +1,28 @@
-// Type definitions for secure-random-password 0.1
+// Type definitions for secure-random-password 0.2
 // Project: https://github.com/rackerlabs/secure-random-password
 // Definitions by: Bjørnar Snoksrud <https://github.com/bjornars>
+//                 GaspardFRDev <https://github.com/GaspardFRDev>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export const lower: string;
-export const upper: string;
 export const consonants: string;
-export const vowels: string;
+export const copyableSymbols: string;
 export const digits: string;
-export const symbols: string;
 export const fullSymbols: string;
-export const copyableSymbols: string;
+export const lower: string;
+export const symbols: string;
+export const upper: string;
+export const vowels: string;
+
+export interface RandomPasswordCharactersSet {
+    characters: string;
+    exactly?: number;
+}
 
 export interface RandomPasswordOptions {
+    avoidAmbiguous?: boolean;
+    characters?: string | Array<(RandomPasswordCharactersSet | string)>;
```

Comment:
It can be an array of `RandomPasswordCharacterSet` but not an individual one?

Reply:
It seems to work with an individual one but it's more straightforward to do : 
`randomPassword({ characters: srp.upper, length: 5})` 
instead of 
`randomPassword({ characters: { characters: srp.upper }, length: 5}})` 
altough you can still use one with array :
`randomPassword({ characters: [{ characters: srp.upper }], length: 5}})`

Reply:
I would lean towards matching the underlying library in this case (i.e. by allowing an individual `RandomPasswordCharacterSet`), but I don't feel strongly about it.